rpki-validator/src/main/java/net/ripe/rpki/validator3/util/ForkJoin.java

```diff
@@ -29,22 +29,47 @@
  */
 package net.ripe.rpki.validator3.util;
 
+import lombok.extern.slf4j.Slf4j;
+
 import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.Semaphore;
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.Supplier;
 
+@Slf4j
 public class ForkJoin {
+
+    private static final Semaphore maximumManagedBlockers;
+
+    static {
+        int maximumSpares = Integer.parseInt(System.getProperty("java.util.concurrent.ForkJoinPool.common.maximumSpares", "256"));
+        int max = Math.min(2 * ForkJoinPool.getCommonPoolParallelism(), maximumSpares / 4);
```

Comment:
Why maximumSpares / 4? 
I would want a comment with a link to some reasoning about these choices.